mentors/java/k.json

```
@@ -31,4 +31,12 @@
     "avatar_url": null,
     "bio": "I have been writing Java professionally for 12 years."
   }
+  {
+     "github_username": "kchatr",
```

```suggestion
    "github_username": "kchatr",
```
One too many spaces.